src/components/Options/TimelineOptions.js

```diff
@@ -0,0 +1,16 @@
+const IsEnabledOptions = [
+  { value: "on", label: "ON" },
```

Comment:
Why not boolean for the `value`?

Reply:
Yes its better to use boolean , updated to boolean